pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -65,33 +96,40 @@ def pre_optimizer_step(
             self.scaler.update()
         return False
 
+    @property
+    def autocast(self) -> torch.cuda.amp.autocast:
```

Comment:
if torch.cuda.amp.autocast() is a function then this should probably also be, no?

Reply:
yep you're right! this looks weird to me, and what you suggest makes more sense :)

Reply:
@awaelchli does `autocast_context_manager` make sense?